
---
bool success
geometry_msgs/Pose distanceToGo
geometry_msgs/Pose position